From 7e213371499499eac50f59e99ef42ec0e7b61da1 Mon Sep 17 00:00:00 2001
From: Gerald Pfeifer <gerald@pfeifer.com>
Date: Sat, 18 Jul 2020 12:02:58 +0300
Subject: [PATCH] ntdll: Include <signal.h> for sigset_t

commit 7053b7c615b44e7112ab8fb4c056206d32b6f1c9
Author: Alexandre Julliard <julliard@winehq.org>
Date:   Fri Jul 17 11:46:15 2020 +0200

    ntdll: Move the registry system calls to the Unix library.

    Signed-off-by: Alexandre Julliard <julliard@winehq.org>

broke non-Linux platforms since sigset_t isn't necessarily defined
based on the #include statements in the new dlls/ntdll/unix/registry.c.

This fixes it.

Observed and tested on FreeBSD 11.4/i386.

Gerald

Signed-off-by: Gerald Pfeifer <gerald@pfeifer.com>
---
 dlls/ntdll/unix/registry.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dlls/ntdll/unix/registry.c b/dlls/ntdll/unix/registry.c
index f94c8ff98ee..6d583618180 100644
--- a/dlls/ntdll/unix/registry.c
+++ b/dlls/ntdll/unix/registry.c
@@ -25,6 +25,7 @@
 #pragma makedep unix
 #endif
 
+#include <signal.h>
 #include <stdarg.h>
 #include <string.h>
 
-- 
2.27.0

